configs/default/SJET-AOCODAF405V3.config

```diff
@@ -124,10 +124,6 @@ serial 3 8192 115200 57600 0 115200
 serial 4 1 115200 57600 0 115200
 
 # master
-set pinio_config = 129,129,1,1
-set pinio_box = 40,41,42,255
-set serialrx_provider = CRSF
-set gps_provider = UBLOX
```

Comment:
Do not remove pinio settings - maybe you misread as reviews shows more lines then intended.

Reply:
Only remove UBLOX here.

Reply:
So just remove serial rx and gps, am i right?

Reply:
serial can be useful for >= 4.3.2